### PR TITLE
Status get/update logic using local state

### PR DESCRIPTION
**Pull request by MaksimkaSota**
1. Added status get/update logic;
2. Added local state for ProfileStatus (useState);

Server errors will be handled at a later stage of development!

I already have isFetchingUsers, isFetchingProfile, isFetchingStatus. 
Maybe we should create a separate part of the state for this and name it fetchingReducer?
I will try to implement this later!

### CODE REVIEW

File: src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  };
+
+  const onChangeLocalStatus = (event) => {
+    setLocalStatus(event.target.value);
+  }
+
+  return (
+    <div className={classes.statusBlock}>
+      {
+        !editMode ?
```
Comment (ZhekaVasil) [inline src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js]: In general such conditions usually starts with positive value, it is easier to read =)
ex:
editMode ? code1 : code2

File: src/redux/thunks/profile.js [hunk 2]
```diff
@@ -9,3 +9,29 @@ export const getProfile = (id) => {
     dispatch(toggleIsFetchingProfile(false));
   };
 };
+
+export const getStatus = (id) => {
+  return async (dispatch) => {
+    const data = await getStatusAPI(id);
+    dispatch(setStatus(data));
+  };
```
Comment (ZhekaVasil) [inline src/redux/thunks/profile.js:17]: I hope you will add error handler here as well in future =)

File: src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js (hunk 1 above)
Comment (MaksimkaSota) [reply src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js]: Done!

File: src/redux/thunks/profile.js (hunk 2 above)
Comment (MaksimkaSota) [reply src/redux/thunks/profile.js:17]: Server errors will be handled at a later stage of development!

File: src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <b className={classes.title}>Status:</b>
+            {
+              isFetchingStatus ?
+                <Preloader className={classes.statusPreloader} /> :
+                <p className={classes.statusText} onDoubleClick={onActivateEditMode}>{status || 'no status'}</p>
+            }
+          </div>
+      }
+    </div>
+  );
```
Comment (ZhekaVasil) [inline src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js]: As I see your have
```
<div className={classes.status}>
            <b className={classes.title}>Status:</b>
```
in both conditions, I can propose the change: 
```
return (
    <div className={classes.statusBlock}>
      <div className={classes.status}>
        <b className={classes.title}>Status:</b>
        {editMode ? (
          <textarea
            autoFocus
            className={classes.inputStatus}
            onBlur={onDeactivateEditMode}
            onChange={onChangeLocalStatus}
            value={localStatus}
          />
        ) : (
          (isFetchingStatus && <Preloader className={classes.statusPreloader} />) || (
            <p className={classes.statusText} onDoubleClick={onActivateEditMode}>
              {status || 'no status'}
            </p>
          )
        )}
      </div>
    </div>
  );
```

Comment (MaksimkaSota) [reply src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js]: Added changes, but used a different code structure!

**Review by ZhekaVasil — APPROVED** [review]

**Pull request merged**